blueocean-dashboard/src/main/js/creation/github/GithubCreationApi.js

```diff
@@ -0,0 +1,77 @@
+import { Fetch } from '@jenkins-cd/blueocean-core-js';
+
+function delayedPromise(value, delay = 2000) {
+    return new Promise(resolve => {
+        setTimeout(() => {
+            resolve(value);
+        }, delay);
+    });
+}
+
+export default class GithubCreationApi {
+
+    constructor(fetch) {
+        this._fetch = fetch || Fetch.fetchJSON;
+    }
+
+    listOrganizations() {
+        return delayedPromise([
```

Comment:
Yes, all of the Github stuff is test code. I'm going to disable Github entirely in this branch since it doesn't do anything (yet).